ansible/roles/logrotate/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - logrotate
```

Comment:
Nice, will this actually run?

Reply:
Good question. I'm not sure. I just  used `ansible-galaxy init logrotate` to initialize the role